dd-java-agent/agent-profiling/profiling-ddprof/src/main/java/com/datadog/profiling/ddprof/DatadogProfiler.java

```diff
@@ -60,11 +60,28 @@ private static final class Singleton {
     private static final DatadogProfiler INSTANCE = newInstance();
   }
 
+  private static void logFailedInstantiation(Throwable error) {
+    if (log.isDebugEnabled()) {
+      log.warn(
+          "failed to instantiate Datadog profiler on {} {}",
+          OperatingSystem.current(),
+          Arch.current(),
+          error);
```

Comment:
The format string specifies only 2 parameters.

```suggestion
      log.warn(
          "failed to instantiate Datadog profiler on {} {}",
          OperatingSystem.current(),
          Arch.current());
```